app/main/posts/views/filters/filters-dropdown.directive.js

```diff
@@ -39,8 +49,15 @@ function FiltersDropdown(PostFilters, ModalService, $rootScope) {
             // @TODO Prevent the user from creating one if they somehow manage to get to this point without being logged in
             $scope.savedSearch.user_id = $rootScope.currentUser ? $rootScope.currentUser.userId : null;
             ModalService.openTemplate('<saved-search-editor saved-search="savedSearch"></saved-search-editor>', 'set.create_savedsearch', 'star', $scope, false, false);
-
-            //ModalService.openTemplate('<savedsearch-create></savedsearch-create>', 'nav.saved_searches', '/img/iconic-sprite.svg#star', $scope.$parent, true, true);
+        };
+        $scope.editSavedSearchModal = function () {
+            SavedSearchEndpoint.get({id: PostFilters.getModeId()}, function (savedSearch) {
```

Comment:
Doing this so we don't directly depend on the saved search object being set in filters and use the modeId  to get the full object from the backend